packages/test/test-end-to-end-tests/src/test/summaryDataStoreStats.spec.ts

```diff
@@ -86,10 +86,18 @@ describeNoCompat("Generate Summary Stats", (getTestObjectProvider) => {
      * synchronizes all containers and waits for a summary that contains the last processed sequence number.
      * @returns the sequence number of the summary
      */
-     async function waitForSummary(): Promise<number> {
-        await provider.ensureSynchronized();
+     async function waitForSummary(timeout: number): Promise<number> {
+        // create the timeout error message since the timeout reason in local test is still not clear
+        // (Bug 1556 on sprint board)
```

Comment:
You could just remove this line. Otherwise, Taylor's guidance for referring to ADO issues in code:

referencing ADO items in GH todos is fine, but it should be over the form "TODO:ADO:[issue number]: [description]".
 
The description should be self-contained and not require access to ADO, allowing external collaborators to get the information they need but still allowing MS to keep scheduling information in ADO.

Reply:
Minus the "TODO" part

Reply:
I just removed the ADO work item info in comments